lib/Runtime/Library/JavascriptNumber.cpp

```diff
@@ -585,19 +585,23 @@ namespace Js
         {
             //use the first arg as the fraction digits, ignore the rest.
             Var aFractionDigits = args[1];
+            bool noRangeCheck = false;
 
             // shortcut for tagged int's
             if(TaggedInt::Is(aFractionDigits))
             {
                 fractionDigits = TaggedInt::ToInt32(aFractionDigits);
             }
             else if(JavascriptOperators::GetTypeId(aFractionDigits) == TypeIds_Undefined)
-                ; // fraction digits = -1
+            {
```

Comment:
There was no point checking for taggedInt, the ToInteger is doing the same thing. Anyway - this is good to go.


---
In reply to: [100207392](https://github.com/Microsoft/ChakraCore/pull/2497#discussion_r100207392) [](ancestors = 100207392,100205438)